content/blog/kubernetes-monitoring-using-prometheus-and-grafana-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Adding a monitoring stack to a Kubernete cluster using Prometheus and
```

Comment:
Hello, Guoping!
I'm so sorry it's taken me so long to get to reviewing your posts. We're in the midst of getting out a new interactive eZine for the organization and it's been a lot of work.  In any case, trying to get to as many reviews as I can today and tomorrow.

In your title, you're missing the "s" on Kubernetes. You might want to fix that. :-)

Reply:
No worries, Dale !